### ISSUE TITLE
Spotify is not viable java.lang.NullPointerException

### ISSUE THREAD

**Issue by sixstone-qq** [janbar/noson-app#90]
Hi!!

After an update from noson 3.13, Spotify is not able to browse anymore even though searching and playing an specific album/playlist works.

![image](https://user-images.githubusercontent.com/440508/65912117-f6354d80-e3cd-11e9-84a2-5c7fb3761832.png)

```
qml: LANG=en_GB
...
qml: Set queue position view at 4
(SONOS)SSLSessionFactory: SSL engine initialized
(SONOS)WSResponse: server error (500)
(SONOS)WSResponse: server error (500)
qml: Fault: java.lang.NullPointerException
```

Launching using `LANG=C noson-app` worked. Let me know if I can help you further on this issue :)

Thanks a lot for your project!! I love it!

**Comment by janbar** [comment]
That is fixed from noson-app release 3.14.5 and up. You should install the latest release of the app: 3.15.0

**Comment by sixstone-qq** [comment]
I'm using noson-app 3.15 (libnoson 1.15.1) using PPA repository for Ubuntu:
```
$ apt show noson-app
Package: noson-app
Version: 3.15~201909282045~ubuntu18.04.1
Priority: extra
Section: misc
```

**Comment by janbar** [comment]
Could you send me to my email jlbarriere68@gmail.com , the debug log by running the app as the following:

noson-app --debug 2>&1 | tee noson.log

**Comment by janbar** [comment]
Thanks for the log file.
Unbelievable ! It seems the Spotify server does no longer support the language string "en-GB" or "en-gb". But it support "en-XX" "en-AA" "en-BB" ... and "en-US" and "en" (without the territory variant).

**Comment by janbar** [comment]
The sample from www.w3.org to explain the language preferences (https://www.w3.org/International/questions/qa-lang-priorities.en) show the following.

`Accept-Language: da, en-gb;q=0.8, en;q=0.7`

**Comment by janbar** [comment]
It should be fixed with the new release 3.15.1. Please let me know.

**Comment by sixstone-qq** [comment]
Yay it worked like a charm :)
